examples/IRMQTTServer/IRMQTTServer.ino

```diff
@@ -1283,6 +1291,20 @@ void loop(void) {
     lastIrReceivedTime = millis();
     lastIrReceived = String(capture.decode_type) + "," +
         resultToHexidecimal(&capture);
+    if (capture.decode_type == -1) {
```

Comment:
replace `-1` with `UNKNOWN`

Reply:
Also, on second thoughts, as we are sending this to MQTT and other people may have implemented stuff using the old expected behaviour. Lets put this extra behaviour all behind some #define switch that the user needs to change.
e.g
`#define REPORT_RAW_UNKNOWN false`

or move it to after `mqtt_client.publish(MQTTrecv, lastIrReceived.c_str());` so it is displayed in HTML only.

The raw line can produce a lot of output, and unless people have a large `MQTT_MAX_PACKET_SIZE` size set, we could hit that limit.

Reply:
Fixed using the enum `UNKNOWN` - thanks for the hint.

Made sending the whole buffer optional with `#define REPORT_RAW_UNKNOWN false` - good idea.

Added warnings to increase `MQTT_MAX_PACKET_SIZE` - very helpful.